config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -344,6 +342,58 @@ presubmits:
           requests:
             memory: "6Gi"
 
+  # clone of pull-perf-tests-clusterloader2 with enabled node killer and using only load test
```

Comment:
Why do we want presubmit too?

Reply:
I'm generally against adding presubmits for experimental things - we can run postsubmits more frequently...

Reply:
It's a manually triggered presubmit - there is a contribution work happening around Node Killer and I prefer to have presubmit to gate those changes, than merge those commits and check for results after that.

Please note that this will only affect PRs that will opt in for this test.

Reply:
I'm not a fan, but I can probably live with it.